src/helpers/shared/helpers.js

```diff
@@ -40,3 +37,26 @@ export const selectedRows = (newSelection, isSelected) => (selected) => {
 };
 
 export const firstUpperCase = (text) => text.charAt(0).toUpperCase() + text.slice(1);
+
+export const createQueryParams = (params) => {
+  const searchParams = new URLSearchParams();
+  Object.keys(params).forEach((key) => {
+    const value = params[key];
+
+    if (Array.isArray(value)) {
+      value.forEach((item) => item && searchParams.append(key, item));
+    } else {
+      value && searchParams.set(key, value);
+    }
+  });
+  return `?${searchParams.toString()}`;
+};
+
+export const getBackRoute = (pathname, pagination, filters) => ({
+  pathname,
+  search: createQueryParams({
```

Comment:
The `createQueryParams` returns a string with `?` prefix. That will cause the router url will contain `??` query prefix. The function should not return it.

Reply:
@Hyperkid123 surprisingly, there was just one `?` - it was working as well. The react-router is probably handling it somehow but did not find exactly where. It's definitely better to delete it, thank you.